shared/common-adapters/button.js.flow

```diff
@@ -7,11 +7,17 @@ export type Props = {
   label: ?string,
   style?: ?Object,
   primary?: ?bool,
+  secondary?: ?bool,
+  follow?: ?bool,
```

Comment:
Should we use an enum (string) for these button types?

I don't think we should have types here for following/unfollow

Also why ?bool and not bool  ... we dont want these values to be true,false,null